### PR TITLE
Update STaaS.md - Troubleshooting

### PR DESCRIPTION
**Pull request by joaobarraca**

### CODE REVIEW

File: user-guide/Troubleshooting/Procedures/Troubleshooting_STaaS_Issues.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+## STaaS is unreachable
+
+If STaaS is unreachable for some reason, the following error message will be shown in the Alarm Console:
+
+![Error in Alarm Console](~/user-guide/images/STaaS_troubleshooting_Alarm_Console.png)
+
+DataMiner will automatically initiate offload mode, ensuring that no data loss occurs. While this error is displayed in the Alarm Console, **do not restart** the system or any of its elements.
+
+This issue will typically resolves itself. If it persists, contact support.
```
Comment (MariekeGO) [inline user-guide/Troubleshooting/Procedures/Troubleshooting_STaaS_Issues.md:86]: Which address should they contact? staas@dataminer.services?

Comment (joaobarraca) [reply user-guide/Troubleshooting/Procedures/Troubleshooting_STaaS_Issues.md:86]: yes 👍

**Comment by MariekeGO** [comment]
@joaobarraca, I've reviewed the PR and made some changes. Could you check if everything is still OK for you?

**Comment by joaobarraca** [comment]
> @joaobarraca, I've reviewed the PR and made some changes. Could you check if everything is still OK for you?

yes, looks good on our side

**Review by MariekeGO — APPROVED** [review]

**Pull request merged**